src/pipecat/services/simli.py

```diff
@@ -0,0 +1,116 @@
+import asyncio
+
+from pipecat.frames.frames import (
+    Frame,
+    OutputImageRawFrame,
+    TTSAudioRawFrame,
+    StartInterruptionFrame,
+    EndFrame,
+    CancelFrame,
+)
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+
+import numpy as np
+from av import AudioFrame
+from av.audio.resampler import AudioResampler
```

Comment:
no need to use ffmpeg resample. we have a `resample_audio` function that we can use.